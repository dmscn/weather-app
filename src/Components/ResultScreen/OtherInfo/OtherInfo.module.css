.Container {
  display: flex;
  align-self: center;
  flex-direction: column;
  justify-content: center;
  font-weight: 200;
}