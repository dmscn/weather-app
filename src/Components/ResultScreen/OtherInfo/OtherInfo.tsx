import React, { Component } from 'react'

import styles from './OtherInfo.module.css';

export default class OtherInfo extends Component {
  render() {
    return (
      <section className={styles.Container}>
        OtherInfo
      </section>
    )
  }
}
