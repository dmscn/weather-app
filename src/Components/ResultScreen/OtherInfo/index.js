export {default} from './OtherInfo';
export * from './OtherInfo';