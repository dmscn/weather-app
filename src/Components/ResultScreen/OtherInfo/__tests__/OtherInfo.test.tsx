import React from 'react'
import {shallow} from 'enzyme';

import OtherInfo from '../OtherInfo';

describe('<OtherInfo />', () => {
  it('renders', () => {
    const wrapper = shallow(<OtherInfo />);
    expect(wrapper).toMatchSnapshot();
  })
});