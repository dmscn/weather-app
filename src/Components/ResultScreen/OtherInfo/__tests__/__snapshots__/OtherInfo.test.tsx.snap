// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<OtherInfo /> renders 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <OtherInfo />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": "OtherInfo",
      "className": "Container",
    },
    "ref": null,
    "rendered": "OtherInfo",
    "type": "section",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": "OtherInfo",
        "className": "Container",
      },
      "ref": null,
      "rendered": "OtherInfo",
      "type": "section",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
